### ISSUE TITLE
3.7.0 Bug: admin email settings not saving

### ISSUE THREAD

**Issue by BusinessBloomer** [woocommerce/woocommerce#24620]
## Steps to reproduce the issue

1. go to  /wp-admin/admin.php?page=wc-settings&tab=email
2. click on the New order email
3. toggle the enable/disable checkbox
4. save
5. checkbox saves temporarily, but then if I go back to /wp-admin/admin.php?page=wc-settings&tab=email nothing has actually happened

## WordPress Environment

WordPress 5.2.3
WooCommerce 3.7.0
Any theme
No other plugin

**Comment by EdithAllison** [comment]
Hi Rodolfo, is your problem linked to [Emoji in "Additional content" field makes it impossible to save changes to WooCommerce emails #24397](https://github.com/woocommerce/woocommerce/issues/24397)?

**Comment by BusinessBloomer** [comment]
> Hi Rodolfo, is your problem linked to [Emoji in "Additional content" field makes it impossible to save changes to WooCommerce emails #24397](https://github.com/woocommerce/woocommerce/issues/24397)?

Yep, that did the trick (I removed the emoji and was able to save)! Thank you
